index.js

```diff
@@ -20,35 +22,238 @@ class J5sExecutor extends Executor {
      */
     _jenkinsCommand(options, callback) {
         // To pass arguments as an array, we need to use apply
-        options.params.push(callback);
-
-        return this.jenkinsClient[options.module][options.action].apply(
+        this.jenkinsClient[options.module][options.action].apply(
```

Comment:
@tkyi This method returns void, so I think it should not return a value explicitly.